### PR TITLE
Pgiraud crosshair map distortion

### PR DESCRIPTION
**Pull request by pgiraud**
#653 needs to be merged first.

I had a lot of troubles fixing this one with the help of @elemoine.
To summarize, we need to wait for ngAnimate to be "ready" before we can put observers on animations.
But we also had troubles with weird behavior in Angular which we haven't fully understood.

### CODE REVIEW

**Comment by pgiraud** [comment]
Not ready for review yet.

File: geoportailv3/static/js/locationinfo/locationinfodirective.js [hunk 1]
```diff
@@ -298,6 +313,17 @@ app.LocationinfoController.prototype.showInfoPane_ =
         this['qrUrl'] = this.qrServiceUrl_ + '?url=' + shorturl;
       }, this));
 
+  // we need to wait for angular to be ready to manage animations
+  var dereg = this.scope_.$watch(goog.bind(function() {
+    return this.$animate_.enabled();
+  }, this), goog.bind(function(newVal) {
+    if (newVal) {
+      this.$timeout_(goog.bind(function() {
```
Comment (elemoine) [inline geoportailv3/static/js/locationinfo/locationinfodirective.js]: I think the use of `$timeout` here deserves a comment in the code, even if why it's needed is not fully understood. We should be honest to ourselves :)

**Comment by pgiraud** [comment]
Comment added. Can I merge?

**Comment by elemoine** [comment]
Yes.

**Pull request merged**